pkg/apis/k0s.k0sproject.io/v1beta1/kubeproxy.go

```diff
@@ -30,9 +31,11 @@ const (
 
 // KubeProxy defines the configuration for kube-proxy
 type KubeProxy struct {
-	Disabled           bool   `json:"disabled,omitempty"`
-	Mode               string `json:"mode,omitempty"`
-	MetricsBindAddress string `json:"metricsBindAddress,omitempty"`
+	Disabled           bool                            `json:"disabled,omitempty"`
+	Mode               string                          `json:"mode,omitempty"`
+	MetricsBindAddress string                          `json:"metricsBindAddress,omitempty"`
+	IPTables           map[string]runtime.RawExtension `json:"iptables,omitempty"`
+	IPVS               map[string]runtime.RawExtension `json:"ipvs,omitempty"`
```

Comment:
Well, maybe, but it will be a different behavior: the current implementation will substitute only explicitly provided options but with the defaulting mechanism any option in the `KubeProxy` struct will make the list of the options empty:
```
	if n.KubeProxy == nil {
		n.KubeProxy = DefaultKubeProxy()
	}
```